src/AccessibilityInsights/Modes/ConfigurationModeControl.xaml.cs

```diff
@@ -139,6 +140,15 @@ private void buttonOk_Click(object sender, RoutedEventArgs e)
                 MainWin.UpdateMainWindowConnectionFields();
             }
 
+            if (appSettingsCtrl.SelectedReleaseChannel != Configuration.ReleaseChannel)
+            {
+                // will probably need to add exception handling here
```

Comment:
Waiting on some updates to VersionSwitcherWrapper.ChangeChannel. Will update PR if those go into master before approval.